src/mock.ts

```diff
@@ -51,26 +61,26 @@ function addMockFunctionsToSchema({ schema, mocks = {}, preserveResolvers = fals
     }
   });
 
-  const defaultMockMap = new Map();
+  const defaultMockMap: Map<string, IMockFn> = new Map();
```

Comment:
`defaultMockMap` is managed internally, and it has only functions.
i think you are refering `mockFunctionMap`, and if that's the case,  then note that it's not possible at the moment:
```typescript
      if (mockFunctionMap.has(fieldType.name)) {
        // the object passed doesn't have this field, so we apply the default mock
        return mockFunctionMap.get(fieldType.name)(root, args, context, info);
      }
```